apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V37__api_progress_table.sql

```diff
@@ -0,0 +1,11 @@
+/*
+ * Creates a table to track the highest sequence number imported into the database.
+ * A query of this table can be used to track the sequence number to pass as a
+ * {@code since} parameter to the RDA API.
+ */
+
+create table "pre_adj"."RdaApiProgress" (
+    "claimType"          VARCHAR(20) NOT NULL PRIMARY KEY,
+    "lastSequenceNumber" BIGINT      NOT NULL,
```

Comment:
with this addition, do we need to keep indexes for the sequence numbers in the FISS and MCS claims tables?

Reply:
Good catch.  I will update the migration script to drop those indexes.

Reply:
FYI - the migration file had to be renamed following a merge because it had the same version number as a migration from the merge.